### PR TITLE
Added reference on how to get data

### PR DESCRIPTION
**Pull request by wasade**
Accessions and a mention of the new Github repo

### CODE REVIEW

File: www/americangut/FAQ.psp [hunk 1]
```diff
@@ -157,6 +158,18 @@
...
+                            <ol>
+                                <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003819&display=html">ERP003819</a></li>
+                                <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003822&display=html">ERP003822</a></li>
+                                <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003820&display=html">ERP003820</a></li>
+                                <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003821&display=html">ERP003821</a></li>
+                            </ol>
+                            <p>Processed sequence data and open-access descriptions of the bioinformatic processing can be found at our <a href="https://github.com/qiime/American-Gut">Github page</a>.</p>
```
Comment (ElDeveloper) [inline www/americangut/FAQ.psp]: `Github page` -> `GitHub repository`.

**Comment by ElDeveloper** [comment]
Just one ultra minor comment and:

Perhaps worth adding a note that all of these resources will be updated as more information is added, right now the link to the repository takes you to an empty repo (with an open pull request though :beers: ).

**Comment by wasade** [comment]
responded to both, I think its good now

**Comment by ElDeveloper** [comment]
Thanks!

**Pull request merged**